test/controllers/defineSignatoryInfo.controller.test.ts

```diff
@@ -233,6 +237,8 @@ describe('DefineSignatoryInfoController', () => {
         .send(form)
         .expect(BAD_REQUEST)
 
+      verify(validator.validate(deepEqual(form), anything())).called()
```

Comment:
Prefer if we're more specific how often it should be called

```suggestion
      verify(validator.validate(deepEqual(form), anything())).once()
```

Reply:
done